client/user/test/mealorder/meal-order-client-app_test.html

```diff
@@ -31,17 +31,21 @@
     <script src="../../bower_components/web-component-tester/browser.js"></script>
     <script src="../../node_modules/redux/dist/redux.js"></script>
     <link rel="import" href="../../src/mealorder/meal-order-client-app.html">
-
+    <link rel="import" href="../../src/mealorder/menu/meal-order-menu-container.html">
+    <link rel="import" href="../../src/mealorder/menu/meal-order-drawer.html">
 </head>
 <body>
 
-<test-fixture id="BasicClientApp">
+<test-fixture id="MealOrderClientApp">
     <template>
         <meal-order-client-app></meal-order-client-app>
     </template>
 </test-fixture>
 
 <script>
+    /*global MealOrderDrawer, MealOrderMenuContainer*/
```

Comment:
Please use application namespace to use component classes outside of component file.